project_remaining_time_on_task/models/project.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    has_contract_hours = fields.Boolean(string="Has contract hours",
+                                        compute="_compute_has_contract_hours")
+
+    remaining_contract_hours = fields.Float(
+        string="Remaining Contract Hours",
+        compute='_compute_remaining_hours_contract',
+        digits=(16, 2))
+
+    def _get_qty_invoiced(self):
+        _qty_invoiced = 0
```

Comment:
The convention `_variable_name` is used to indicate that a variable in a class is private. No need for that prefix in a method.

Reply:
ok